lib/teleterm/api/proto/v1/service.proto

```diff
@@ -287,4 +290,21 @@ message GetAuthSettingsRequest {
   string cluster_uri = 1;
 }
 
+message ClusterEventsRequest {}
+
+message ClusterEvent {
+  string cluster_uri = 1;
+  oneof event {
+    CertExpired cert_expired = 2;
+    NewGatewayConnectionAccepted new_gateway_connection_accepted = 3;
+  }
+}
```

Comment:
This is just a draft version of this message. For now I'm fairly sure that all messages will be cluster-centric. Though rewriting them to support messages that are not tied to any particular cluster shouldn't be hard.